script.js

```diff
@@ -0,0 +1,45 @@
+// Set variables for each respective move
+var x = "X"
+var o = "O"
+// Create functions to place X or O, taking in boxNumber to choose where to place
+function placeX(boxNumber) {
+	document.getElementById(boxNumber).innerHTML = x;
+}
+function placeO(boxNumber) {
+	document.getElementById(boxNumber).innerHTML = o;
+}
+
+// Set variables to easily select individual cells and determine which player's turn it is
+
+var cells = document.getElementsByTagName("TD");
+var playerTurn = document.getElementsByClassName("player-turn")[0];
```

Comment:
An ID may be more appropriate here, since there should only be one element on this page related to current player's turn.